<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiChannelSettings</class>
 <widget class="QDialog" name="uiChannelSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>603</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>603</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ustawienia kanałów</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/logo_64.png</normaloff>:/images/logo_64.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>561</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ogólne</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>537</width>
       <height>531</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_owner">
        <property name="text">
         <string>Właściciel:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_created">
        <property name="text">
         <string>Utworzono:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_website">
        <property name="text">
         <string>Strona kanału:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_topic">
        <property name="text">
         <string>Temat:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_desc">
        <property name="text">
         <string>Opis:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_status">
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_category">
        <property name="text">
         <string>Kategoria:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_guardian">
        <property name="text">
         <string>Anioł stróż:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>Hasło:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_limit">
        <property name="text">
         <string>Limit nicków:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_moderation">
        <property name="text">
         <string>Moderacja:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_auditorium">
        <property name="text">
         <string>Audytorium:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_owner_nick">
          <property name="text">
           <string>Unknown</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_transfer">
          <property name="text">
           <string>Przekaż</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_remove_channel">
          <property name="text">
           <string>Usuń</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="comboBox_category">
        <item>
         <property name="text">
          <string>Teen</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Towarzyskie</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Erotyczne</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tematyczne</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Regionalne</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="10" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEdit_password"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_set_password">
          <property name="text">
           <string>Ustaw</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="1">
       <widget class="QSpinBox" name="spinBox_limit"/>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLineEdit" name="lineEdit_website"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_set_website">
          <property name="text">
           <string>Ustaw</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_topic">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>301</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>301</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_set_topic">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ustaw</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_desc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>301</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>301</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_set_desc">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ustaw</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="1">
       <widget class="QWidget" name="widget_6" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>373</width>
          <height>44</height>
         </size>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioButton_status_pub">
            <property name="text">
             <string>Publiczny</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_status_priv">
            <property name="text">
             <string>Prywatny</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>430</width>
          <height>43</height>
         </size>
        </property>
        <widget class="QLabel" name="label_datetime">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>430</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>430</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>Data/Czas</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QWidget" name="widget_11" native="true">
        <property name="maximumSize">
         <size>
          <width>373</width>
          <height>43</height>
         </size>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QRadioButton" name="radioButton_moderation_off">
            <property name="text">
             <string>Wyłączona</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_moderation_on">
            <property name="text">
             <string>Włączona</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QWidget" name="widget_12" native="true">
        <property name="maximumSize">
         <size>
          <width>373</width>
          <height>44</height>
         </size>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QRadioButton" name="radioButton_auditorium_off">
            <property name="text">
             <string>Wyłączone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_auditorium_on">
            <property name="text">
             <string>Włączone</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QWidget" name="widget_8" native="true">
        <property name="maximumSize">
         <size>
          <width>373</width>
          <height>43</height>
         </size>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>371</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButton_guardian_off">
            <property name="text">
             <string>Wyłączony</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_guardian_on">
            <property name="text">
             <string>Włączony</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_guardian_level">
            <item>
             <property name="text">
              <string>Poziom 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Poziom 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Poziom 3</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_email">
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLineEdit" name="lineEdit_email"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_set_email">
          <property name="text">
           <string>Ustaw</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_channel">
        <property name="text">
         <string>Kanał:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_channel_name">
        <property name="text">
         <string>Channel name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Uprawnienia</string>
    </attribute>
    <widget class="QToolBox" name="toolBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>531</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Super operatorzy</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>461</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget_op">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>1</number>
          </property>
          <column>
           <property name="text">
            <string>Nick</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="pushButton_op_add">
            <property name="text">
             <string>Dodaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_op_del">
            <property name="text">
             <string>Usuń</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Operatorzy</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>471</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget_halfop">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>1</number>
          </property>
          <column>
           <property name="text">
            <string>Nick</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="pushButton_halfop_add">
            <property name="text">
             <string>Dodaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_halfop_del">
            <property name="text">
             <string>Usuń</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>561</width>
        <height>413</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Wyproszeni</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>461</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="tableWidget_ban">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <column>
           <property name="text">
            <string>Nick</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Założył</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Data/Czas</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QPushButton" name="pushButton_ban_add">
            <property name="text">
             <string>Dodaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_ban_del">
            <property name="text">
             <string>Usuń</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Zaproszeni</string>
      </attribute>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>461</width>
         <height>401</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidget_invite">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <column>
           <property name="text">
            <string>Nick</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Założył</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Data/Czas</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QPushButton" name="pushButton_invite_add">
            <property name="text">
             <string>Dodaj</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_invite_del">
            <property name="text">
             <string>Usuń</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>570</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
